README.md

```diff
@@ -1,6 +1,15 @@
-# Our micro blogging APP, the sky is the limit!!!
+# Tiny Thoughts microblogging app
 
+## Our microblogging APP, the sky is the limit!!!
 
+## How to run the app
```

Comment:
If this could be changed to how to run the tests in Cypress, it would be clearer  👍🏼